draft-ietf-acme-rats.md

```diff
@@ -143,19 +144,18 @@ The claims and mechanisms defined in {{RATSPA}} are a good basis for the assessm
 
 This is a new identifier type.
 
-type (required, string):
-: The string "trusthworthy".
-
-value (required, string):
-: The constant string "trustworthy"
+* type (required, string): The string "trusthworthy".
+* value (required, string): The constant string "trustworthy"
 
 The following sections detail the changes.
 
+EDNOTE: why not use "attested" or "rats"?
```

Comment:
1. "attested" is vague in the industry.
2. "rats" is the name of the WG, not a protocol or thing.

In the end, the string is arbitrary.

Reply:
I agree that the string is arbitrary. I guess I don't like "trustworthy" because many things are trustworthy, including but not limited to things that do RATS style attestation.

I don't really care what string we choose, but I think it should be something clearly linked to the title of this document, which "trustworthy" is not. "rats" isn't a protocol, but it's the title of RFC9334, which certainly counts as a "thing". Why would we shy away from using it?